src/model/transferlog.js

```diff
@@ -93,7 +134,7 @@ module.exports = class TransferLog {
     }
 
     debug('stored ' + transfer.id, 'isIncoming', isIncoming)
-    yield this._storePackaged({
+    this._storePackaged({
       transfer: transfer,
       state: (transfer.executionCondition ? 'prepared' : 'executed'),
```

Comment:
`_storePackaged` just puts it in the store variable in memory. `write` is the only call that actually writes to disk.